Oswaldo Vazquez Aparicio/26 Agosto/miniproject/Product.java

```diff
@@ -0,0 +1,25 @@
+package miniproject;
+
+public class Product {
+
+    private final float unitPrice;
+    private final int quantity;
+
+    public Product(float unitPrice, int quantity) {
+        this.unitPrice = unitPrice;
+        this.quantity = quantity;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public float getUnitPrice() {
+        return unitPrice;
+    }
+
+    public float getTotalPrice() {
```

Comment:
This method is OK but what about if your method receives parameters like, let's say, unitPrice and quantity... think about it...